guesser.js

```diff
@@ -0,0 +1,24 @@
+function randomNum(input){
+  return Math.floor((Math.random() * input) + 1);
+}
+
+class GuessingGame{
+  constructor(number){
+    this.answer = randomNum(number);
```

Comment:
When we set up the guessing game we want the use to input the answer. so we would just want `this.answer = number`